packages/devtools_app/lib/src/charts/chart.dart

```diff
@@ -600,7 +594,7 @@ class ChartPainter extends CustomPainter {
     tp.paint(canvas, Offset(x + -tp.width / 2, y - tp.height / 2));
   }
 
-  TextPainter createText(String textValue, double scale) {
+  TextPainter createText(String? textValue, double scale) {
```

Comment:
non nullable. The TextSpan widget will throw assertion error if this is null (since semanticsLabel is not specified here either)

Reply:
done